src/components/Marker.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {convertDateToMoment} from '../utils/timeUtils';
+
+/**
+ * A `Marker` is a component draws a vertical line.
+ * @extends React.Component<Marker.propTypes>
+ */
+export class Marker extends React.Component {
+  static propTypes = {
+    /**
+     * The position of the marker, as date (numeric/millis or moment object, cf. `Timeline.props.useMoment`).
+     * @type { object | number }
+     */
+    start: PropTypes.oneOfType([PropTypes.object, PropTypes.number]).isRequired,
```

Comment:
Sa redenumim in "date". Lumea se cam asteapta sa vada si un "end" atunci cand vede "start".

Reply:
Am modificat.